### PR TITLE
SBT publish... and add tests to main itself

### PR DESCRIPTION
**Pull request by arielshaqed**
Added tests were reviewed in #1 but incorrectly merged.

The _real_ diff here is in .sbt files.

### CODE REVIEW

**Comment by arielshaqed** [comment]
I will pull with _any_ reviewer's approval, unless otherwise requested!

File: project/pgp-sign.sbt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+credentials += Credentials(
+  "GnuPG Key ID",
+  "gpg",
+  "F638279A7281EE8EAE58C4B98D3CFE7E7D0262DA", // key identifier
```
Comment (Jonathan-Rosenberg) [inline project/pgp-sign.sbt:4]: is this supposed to be public?

**Review by Jonathan-Rosenberg — APPROVED** [review]
LGTM
Just one question (can merge if it's false)

File: project/pgp-sign.sbt (hunk 1 above)
Comment (arielshaqed) [reply project/pgp-sign.sbt:4]: Yeah, it's confusing and scary!

https://keyserver.ubuntu.com/pks/lookup?search=F638279A7281EE8EAE58C4B98D3CFE7E7D0262DA&fingerprint=on&op=index

It's the public key for lakeFS signing. We do the same in lakeFSFS.

**Review by arielshaqed — COMMENTED** [review]
Thanks for the caution! Much appreciated...

**Pull request merged**